base/applications/screensavers/logon/resource.h

```diff
@@ -1,5 +1,4 @@
 #pragma once
 
 #define IDS_TEXT        3
-#define IDB_WORKSTATION 0x100
-#define IDB_SERVER      0x200
+#define IDB_LOGO        0x100
```

Comment:
This change could potentially make jeffbox unhappy. You know we already have different gina/shell banners for Server vs. Workstation...

Reply:
I could point `IDB_WORKSTATION` and `IDB_SERVER` to the same file and leave a comment in `resource.h` saying to include the new bitmap and update the respective path if jeffbox or someone else would like to change it.

Perhaps we should strive to pull the bitmap from the system. Windows Vista's logon screensaver doesn't include the Windows logo in its resources.

Reply:
I have no problems with this change until we figure out a proper way of handling the banner logic.